lib/libzpool/uzfs_rebuilding.c

```diff
@@ -346,11 +392,12 @@ uzfs_zvol_destroy_snaprebuild_clone(zvol_state_t *zv,
  */
 int
 uzfs_zvol_create_snaprebuild_clone(zvol_state_t *zv,
```

Comment:
should we consider directly populating into zv's member variables rather than function variables?

Reply:
This function is called under zinfo->zinfo_mutex so I think it is okay to live with it.

Reply:
super sir.. probably, we can add in comments of this function that zinfo_mutex lock is required, and an assert would be good